libs/datasets/manual_filter.py

```diff
@@ -96,3 +100,70 @@ def run(dataset: timeseries.MultiRegionDataset, config=CONFIG) -> timeseries.Mul
         dataset = filtered_dataset.append_regions(passed_dataset)
 
     return dataset
+
+
+# Possible values from https://github.com/covid-projections/covid-projections/blob/develop/src/cms-content/region-overrides/region-overrides.json
+_METRIC_TO_FIELDS = {
+    "metrics.caseDensity": common_fields.FIELD_GROUP_TO_LIST_FIELDS[FieldGroup.CASES_DEATHS],
+    # "metrics.infectionRate": common_fields.FieldGroup.
+    "metrics.testPositivityRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[FieldGroup.TESTS],
+    "metrics.icuCapacityRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[
+        FieldGroup.HEALTHCARE_CAPACITY
+    ],
+    "metrics.vaccinationsInitiatedRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[
+        FieldGroup.VACCINES
+    ],
```

Comment:
It seems like your FieldGroup classifications may be a better way of classifying data to be blocked.  Want to propose a set of stringified names we could use in the JSON and I can sanity-check them and maybe I'll move the frontend over to use the same strings (even if it doesn't implement them completely)?  Or do you have other suggestions for a schema/syntax for specifying the fields to be blocked?

Reply:
I don't have that much vision for how this will be used. How about merge and iterate?

Reply:
Okay.  I was mostly just thinking of using the field group names except maybe split cases/deaths, but we can punt for now.